tools/scripts/broker_performance_test.sh

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#!/bin/bash
+
+# deleting existing folders
+if [ ! -e target/ ];
+then
+    mkdir target
+fi
+
+# get the properties file location
+properties_file_location=$1
+
+if [ "$properties_file_location" == '' ]
+        then
+            properties_file_location=resources/ballerina_message_broker_performance_test.properties
+    fi
```

Comment:
Shall we make the indentation consistent across the file?

Reply:
Done